webapp-django/crashstats/crashstats/management/commands/bugassociations.py

```diff
@@ -79,20 +79,18 @@ def add_arguments(self, parser):
         )
 
     def handle(self, **options):
-        start_date = options.get('last_success')
-        if not start_date:
+        start_date_arg = options.get('last_success')
+        if not start_date_arg:
             # Default to now minus a day
             start_date = timezone.now() - datetime.timedelta(days=1)
         else:
             # Try to parse it as a datetime
-            start_date = parse_datetime(start_date)
+            start_date = parse_datetime(start_date_arg)
             if not start_date:
                 # Try to parse it as a date
-                start_date = parse_date(start_date)
+                start_date = parse_date(start_date_arg)
             if not start_date:
-                raise CommandError(
-                    'Unrecognized last_success format: %s' % options.get('last_success')
-                )
+                raise CommandError('Unrecognized last_success format: %s' % start_date_arg)
```

Comment:
I'm pretty sure this code was wrong before in the sense that line 88 stomps on `start_date` if the value isn't in date format. After that, everything is bananas.

I figured I'd fix it before I copy-and-paste the approach for verifyprocessed.